targets/ChibiOS/_nanoCLR/nanoFramework.Hardware.Rpi/nf_hardware_rpi_native_nanoFramework_Hardware_Rpi_Pio_PioStateMachine.cpp

```
@@ -0,0 +1,401 @@
+    PIO_CFG_OUT_EN_SEL = 26,
+    PIO_CFG_BLOB_LENGTH = 27,
+};
+
+void PioStateMachine::NativeInit(
+    signed int param0,
+    signed int param1,
+    signed int param2,
+    CLR_RT_TypedArray_UINT32 param3,
+    HRESULT &hr)
+{
+    PIO_TypeDef *pio = PioFromIndex(param0);
```

Clang (14.0.6)

[warning] variable 'pio' is not initialized

(cppcoreguidelines-init-variables)